entrypoint.sh

```diff
@@ -16,5 +16,8 @@ if [ ! $TOKEN ]; then
     exit 1
 fi
 
-echo -ne 'Y\n\n' | ./config.sh --url $URL --token $TOKEN --name $NAME --work $WORK_DIR
+[ -d _token ] && cp -R _token/. .
```

Comment:
Are the changes in this file related to the docker build pipeline?  Seems like its for something else